notebooks/MetPy_Advanced/Isentropic Analysis.ipynb

```diff
@@ -210,24 +250,37 @@
     "rh = ds['Relative_humidity_isobaric'][0]\n",
     "height = ds['Geopotential_height_isobaric'][0]\n",
     "u = ds['u-component_of_wind_isobaric'][0]\n",
-    "v = ds['v-component_of_wind_isobaric'][0]"
+    "v = ds['v-component_of_wind_isobaric'][0]\n",
+    "\n",
+    "# Due to a different number of vertical levels find where they are common\n",
+    "lev_hght = temperature.coords['isobaric4'] * units.Pa\n",
+    "lev_uwnd = u.coords['isobaric'] * units.Pa\n",
+    "_, _, common_ind = np.intersect1d(lev_uwnd, lev_hght, return_indices=True)\n",
+    "temperature = temperature[common_ind, :, :]\n",
+    "# Reassign units to variables\n",
+    "temperature.attrs['units'] = 'kelvin'\n",
```

Comment:
Is xarray dropping units here or are the original units bad?

Reply:
It's dropping units there.

Reply:
😞 That one *doesn't* seem sensible.